blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Announcing Sourcegraph 3.1'
+author: 'Ryan Blunden'
+publishDate: 2019-02-20T05:59-06:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-3.1
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: false
+---
+
+Sourcegraph 3.1 is now out! This release includes stability and documentation improvements, plus an easy query builder for searching your code (with useful query examples). Also, mark your calendars: our new fixed release schedule means you get a new Sourcegraph release on the 20th of each month.
+
+<!-- (ryan): Hyperlink to headings in above paragraph -->
+
+What is Sourcegraph? Sourcegraph is an open-source, self-hosted, cross-repository code search and navigation tool, with an efficient web interface and "feels-like-native" integration into your code host. [Install or upgrade Sourcegraph.](#install-or-upgrade)
+
+## Find answers in code more easily with the new query builder
+
+<!-- TODO (ryan): Below paragraph needs work as it sounds awkward -->
+
+Did you know that in addition to code, you can search commit diffs, commit messages and symbols?
+
+Even experienced Sourcegraph users will find something new. Give it a try and let us know what you think via [GitHub](https://github.com/sourcegraph/sourcegraph/issues/new/choose) or [Twitter](https://twitter.com/srcgraph).
+
+<!-- TODO (ryan): GIF using the query builder -->
+
+## Improved go-to-definition and hover tooltips for all languages
+
+We've improved Sourcegraph's [code intelligence extensions](https://sourcegraph.com/extensions?query=category%3A%22Programming+languages%22), which let you go to definitions, find references, and see hover tooltips on your code (in all languages).
+
+- The docstrings in hover tooltips are more accurate.
+- Indented sections in docstrings are rendered as code blocks in hovers.
+- Tokens in comments are ignored unless they look like code (e.g., if they're wrapped in backticks or look like a function call `foo()`)
+
+We're actively improving code intelligence in all languages, so please [send us feedback](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=&labels=&template=feature_request.md&title=Language%20extension%20improvement%20for)!```
+
+## Site admin improvements
+
+We've added a couple features to help site admins manage a Sourcegraph instance in an enterprise environment:
+
+- Improved [management console](https://docs.sourcegraph.com/admin/management_console) JSON editor for critical configuration, now with editor completion, inline documentation, and validation.
+- Querying users by email address.
```

Comment:
in addition to @nicksnyder's comments here, remove the period because this is not a sentence (and a similar fragment in the next bullet point lacks a period)